ietf-optical-impairment-topology.tree

```diff
@@ -263,11 +263,11 @@ module: ietf-optical-impairment-topology
           |     +--ro roadm-cd?
           |     |       l0-types:decimal-5-digits-or-null
           |     +--ro roadm-pdl?
-          |     |       l0-types:ratio-in-db-or-null
+          |     |       l0-types:loss-in-db-or-null
           |     +--ro roadm-inband-crosstalk?
-          |     |       l0-types:ratio-in-db-or-null
+          |     |       l0-types:loss-in-db-or-null
```

Comment:
no, inband crosstalk is not a loss, I prefer to keep ratio-in-db-or-null

Reply:
@EstherLerouzic: we have removed  ratio-in-db-or-null, adding 
Gain-in-db
Gain-in-db-or-null
Loss-in-db
Loss-in-db-or-null.

In this case what do you recommend?

Reply:
Addressed in https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/pull/135/commits/62ec3365bb97f7d5b1a2b3135a70091c2f52cbc5

Co-authored-by: sergio belotti <sergio.belotti@nokia.com>